src/stories/PasswordInput.stories.js

```diff
@@ -6,7 +6,7 @@ export default {
   component: PasswordInput
 };
 
-export const Empty = () => <PasswordInput value="" />;
+export const Empty = () => <PasswordInput />;
```

Comment:
Now, you can not control the component anymore which makes it not useful if you want to use it in your app